src/___new___/components/FelixConfig/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import jsonData from './file.json';
+import FelixConfigTable from '../FelixConfigTable';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+const FelixConfig = ({ configType, name }) => {
+  const matchedGroup = jsonData.Groups.find(group => group.Name === name);
+  // Debugging logs
+
+  if (!matchedGroup) {
+    return <p>No matching group found for '{name}'.</p>;
+  } 
+
+
```

Comment:
make sure you format all of your files `option + shift + f`. You may need to install prettier if it doesn't work

Reply:
Done, thanks!